src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function getPartnerSex(sex) {
+  return sex === 'm' ? 'wife' : 'husband';
+}
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">{person.age && `I am ${person.age}`}</p>
```

Comment:
```suggestion
    {person.age && (
      <p className="Person__age">
        {person.age}
      </p>
    )}
```
Don't render `Person__age` paragraph if age is absent